custom_components/frigate/media_source.py

```diff
@@ -1331,25 +1277,53 @@ def _get_recording_base_media_source(
             children=[],
         )
 
-    def _browse_recording_folders(
-        self, identifier: RecordingIdentifier, folders: list[dict[str, Any]]
+    def _get_camera_recording_identifiers(
+        self, identifier: RecordingIdentifier, config: dict[str, dict]
     ) -> BrowseMediaSource:
-        """Browse Frigate recording folders."""
+        """List cameras for recordings."""
         base = self._get_recording_base_media_source(identifier)
 
-        for folder in folders:
-            if folder["name"].endswith(".mp4"):
-                continue
-            title = self._generate_recording_title(identifier, folder)
+        for camera in config["cameras"].keys():
+            base.children.append(
+                BrowseMediaSource(
+                    domain=DOMAIN,
+                    identifier=attr.evolve(
+                        identifier,
+                        **identifier.get_changes_to_set_next_empty(camera),
```

Comment:
Instead of: `get_changes_to_set_next_empty` I think you can just use:

```
identifier=attr.evolve(
  identifier,
  camera=camera)
```

Reply:
Thanks, I honestly wasn't 100% sure what this was used for.